### PR TITLE
Add the "enter ivy folder" step to the "Forking and cloning the repo" section in "Setting Up" docs page

### PR DESCRIPTION
**Pull request by fabriziocacicia**

### CODE REVIEW

**Comment by fabriziocacicia** [comment]
@zaeemansari70 I need a look from you for the wording used

File: docs/overview/contributing/setting_up.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ In order to install and properly set up pre-commit, these steps should be follow
 
 1. Run :code:`python3 -m pip install pre-commit`
 
-2. Enter into your cloned ivy folder, for example :code:`cd ~/ivy`
+2. If you are not alredy into your cloned ivy folder, enter it, for example :code:`cd ~/ivy`
```
Comment (zaeemansari70) [inline docs/overview/contributing/setting_up.rst:61]: No need to change this line, it's pretty much clear. Feel free to un-commit this one.

File: docs/overview/contributing/setting_up.rst [hunk 2]
```diff
@@ -35,7 +35,7 @@ Depending on your preferred mode of cloning, any of the below should work:
 
     gh repo clone YOUR_USERNAME/ivy your_folder -- --recurse-submodules
 
-Then add Ivy original repository as upstream, to easily sync with latest changes.
+Then enter into your cloned ivy folder, for example :code:`cd ~/ivy` and add Ivy original repository as upstream, to easily sync with latest changes.
```
Comment (zaeemansari70) [inline docs/overview/contributing/setting_up.rst:38]: This one looks good to me. :+1:

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Feel free to merge after the changes required :+1:

**Pull request merged**